src/anupSJun20/PhoneNumberValidation.java

```diff
@@ -0,0 +1,76 @@
+package anupSJun20;
+/*Assignment 24 :- 5th Jul'2020 
+Validate given phone number.
++91 9765463742 --> valid
+09765463742 --> valid
+976 546 3742 --> valid
+9765463742 --> valid
+9765463A42 --> Invalid
+976546342 --> Invalid
+6976546342 --> Invalid
+Note : phone number is given in String format.*/
+
+public class PhoneNumberValidation {
+
+	static void findPhoneNumberhavingCharacter(String number) {
+		boolean flag = true;
+		for (int index = 0; index < number.length(); index++) {
+			if (Character.isLetter(number.charAt(index))) {
+				System.out.println("The phone number : " + number + " is invalid ");
+				flag = false;
+			}
```

Comment:
once this if condition pass then need to break loop, do not go for further checking.

Reply:
Added the break statement.